test/srt/openai_server/features/test_enable_thinking.py

```diff
@@ -184,5 +189,55 @@ def test_stream_chat_completion_without_reasoning(self):
         )
 
 
+## Skip for ci test
```

Comment:
Uncomment this?

Reply:
https://github.com/sgl-project/sglang/pull/8445/files